scripts/setup/install_kissat.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# Source kani-dependencies to get KISSAT_VERSION
+source kani-dependencies
+
+if [ -z "${KISSAT_VERSION:-}" ]; then
+  echo "$0: Error: KISSAT_VERSION is not specified"
+  exit 1
+fi
+
+
+# Kissat release
```

Comment:
Can we check if the version is already installed first?

Reply:
Done.